src/components/ebay-donut-chart/index.marko

```diff
@@ -0,0 +1,42 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+$ const {
+    title,
+    metricValue,
+    metricLabel,
+    class: inputClass,
+    series,
+    ...htmlInput
+} = input;
+
+<div class="ebay-donut-chart-container">
+    <div
+        ...processHtmlAttributes(htmlInput)
+        class=[inputClass, "ebay-donut-chart"]
+    >
+        <div class="ebay-donut-chart__title-container">
+            ${title}
+        </div>
+        <div class="ebay-donut-chart__metric-container">
+            <div class="ebay-donut-chart__metric-value">
+                ${metricValue}
+            </div>
+            <div class="ebay-donut-chart__metric-label">
+                ${metricLabel}
+            </div>
+        </div>
+        <div
+            id=component.getContainerId()
+            class="ebay-donut-chart__graph-container"
+            no-update
```

Comment:
Why is there `no-update` here?

Reply:
I noticed the state update (for active portion hover) was causing the highcharts to re-render and destroying the chart instance.

I can work on another solution, as `no-update` would cause issues if there's a need for data to update dynamically